paper-button-behavior.html

```diff
@@ -18,19 +19,36 @@
 
     properties: {
 
-      _elevation: {
-        type: Number
+      /**
+       * The z-depth of this element, from 0-5. Setting to 0 will remove the
+       * shadow, and each increasing number greater than 0 will be "deeper"
+       * than the last.
+       *
+       * @attribute elevation
+       * @type number
+       * @default 1
+       */
+      elevation: {
```

Comment:
Previously `_elevation` was considered an implementation detail of buttons, and not exposed as a public property. Why did you decide to expose `elevation` as a public property?